keepalived-vip/README.md

```diff
@@ -24,19 +24,25 @@ With 2 or more instance of the pod running in the cluster is possible to provide
 This should be considered a complement, not a replacement for HAProxy or nginx. The goal using keepalived is to provide high availability and to bring certainty about how an exposed service can be reached (beforehand we know the ip address independently of the node where is running). For instance keepalived can use used to expose the service-loadbalancer or nginx ingress controller in the LAN using one IP address.
 
 
-
 ## Requirements
 
 [Daemonsets](https://github.com/kubernetes/kubernetes/blob/master/docs/design/daemon.md) enabled is the only requirement. Check this [guide](https://github.com/kubernetes/kubernetes/blob/master/docs/api.md#enabling-resources-in-the-extensions-group) with the required flags in kube-apiserver.
 
 
-
 ## Configuration
 
-To expose one or more services use the flag `services-configmap`. The format of the data is: `external IP -> namespace/serviceName`.
+To expose one or more services use the flag `services-configmap`. The format of the data is: `external IP -> namespace/serviceName`. Optionally is possible to specify forwarding method using `:` after the service name. The valid options are `NAT` and `DR`. For instance `external IP -> namespace/serviceName:DR`.
+By default if the method is not specified it will use NAT.
+
 This IP must be routable inside the LAN and must be available. 
 By default the IP address of the pods are used to route the traffic. This means that is one pod dies or a new one is created by a scale event the keepalived configuration file will be updated and reloaded.
 
+## [Direct Routing](http://www.linuxvirtualserver.org/VS-DRouting.html)
+
```

Comment:
@bprashanth my original goal with this was to fix the source address issues with flannel (the source is not the client IP).
I have another PR after this to use proxy protocol in conjunction with haproxy:
`Internet ->  keepalived VIP -> haproxy with send-proxy`

(this solves the issue with the source ip address)